docs/api/rest_api/rest_api_reference/input/examples/product/catalog/product/code/prices/POST/CustomPriceCreateStruct.json.example

```diff
@@ -0,0 +1,19 @@
+{
+    "CustomPrice": {
```

Comment:
This file should be renamed to `CustomPrice` and moved outside `POST` directory as it doesn't contain any payload sent via POST request.